bot/cogs/watchchannels/talentpool.py

```diff
@@ -65,7 +65,7 @@ async def oldest_command(self, ctx: Context, update_cache: bool = True) -> None:
 
     @nomination_group.command(name='watch', aliases=('w', 'add', 'a'), root_aliases=("nominate",))
     @with_role(*STAFF_ROLES)
-    async def watch_command(self, ctx: Context, user: FetchedMember, *, reason: str) -> None:
+    async def watch_command(self, ctx: Context, user: FetchedMember, *, reason: Optional[str] = '') -> None:
```

Comment:
Im not seeing it. Maybe I do not understand the Optional annotation. `reason: Optional[str]` ? like I thought it was to annotate that it could be "falsy" a string.

Reply:
Nah, `Optional` is explicitly `Union[str, None]`, so the alternative must be a NoneType.